src/components/Privacy/PrivacyLanguages.tsx

```diff
@@ -5,19 +5,23 @@ import { useUserPreferences } from '@context/UserPreferences'
 import Link from 'next/link'
 
 export default function PrivacyLanguages({
```

Comment:
todo: refactor this component to a more generic `PolicyLanguages` component which can be used for cookies, privacy and future policies.